spring-cloud-netflix-core/src/main/java/org/springframework/cloud/netflix/zuul/filters/ZuulProperties.java

```diff
@@ -357,6 +359,26 @@ public boolean isCustomSensitiveHeaders() {
 		
 	}
 
+	@Data
```

Comment:
We are moving away from Lombok so we should not be adding any additional uses of it.